.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
are these allowed in JSON ?

Reply:
"//": "Use IntelliSense to learn about possible attributes."
    "//": "Hover to view descriptions of existing attributes."
    "//": "For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387"
    
    Will update like this, these comments are already in boilerplate template.